keras/layers/recurrent.py

```diff
@@ -38,19 +61,21 @@ def __init__(self, input_dim, output_dim,
         if weights is not None:
             self.set_weights(weights)
 
-    def _step(self, x_t, h_tm1, u):
+    def _step(self, x_t, mask_tm1, h_tm1, u):
```

Comment:
Shouldn't this variable be named `mask_t` everywhere? Or am I missing something?